tdrs-backend/tdpservice/users/migrations/0013_user_sub.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-05-10 14:10
```

Comment:
This migration is autogenerated from the addition of the `UUIDField` to the `User` model.